src/main/java/ch/njol/skript/util/Time.java

```diff
@@ -49,10 +59,28 @@ public int getTicks() {
 	public int getTime() {
 		return (time + HOUR_ZERO) % TICKS_PER_DAY;
 	}
-	
+
+	public int getHour() {
+        int hour = (int) Math2.floor((double) (time + HOUR_ZERO) / TICKS_PER_HOUR);
```

Comment:
```suggestion
        int hour = (time + HOUR_ZERO) / TICKS_PER_HOUR);
```
integer division already truncates